File: assets/js/fibra.js
var nextArrow = '<div class="control-next-arrow">▶▶</div>';
var prevArrow = '<div class="control-prev-arrow">◀◀</div>';
var maxSlidePosition = 9;

var control = {
    0: {
        id: "first-promo",
        title: "Фиброцементный сайдинг: предложение",
        prev: "",
        next: "Виды покрытий" + nextArrow,
    },
    1: {
        id: "summary",
        title: "Фиброцементный сайдинг: обзор",
        prev: prevArrow + "Предложение",
        next: "Палитра цветов" + nextArrow,
    },
    2: {
        id: "palitra",
        title: "Фиброцементный сайдинг: палитра цветов",
        prev: prevArrow + "Обзор",
        next: "Качества" + nextArrow,
    },
    3: {
        id: "kachestva",
        title: "Фиброцементный сайдинг: качества",
        prev: prevArrow + "Палитра цветов",
        next: "Технология монтажа" + nextArrow,
    },
    4: {
        id: "schema",
        title: "Фиброцементный сайдинг: монтаж",
        prev: prevArrow + "Качества",
        next: "Устройство откосов" + nextArrow,
    },
    5: {
        id: "otkos",
        title: "Фиброцементный сайдинг: откосы",
        prev: prevArrow + "Технология монтажа",
        next: "Портфолио" + nextArrow,
    },
    6: {
        id: "portfolio",
        title: "Фиброцементный сайдинг: портфолио",
        prev: prevArrow + "Устройство откосов",
        next: "Видео обзор" + nextArrow,
    },
    7: {
        id: "video",
        title: "Фиброцементный сайдинг: видео обзор",
        prev: prevArrow + "Портфолио",
        next: "Предложение" + nextArrow,
    },
    8: {
        id: "credit-promo",
        title: "Фиброцементный сайдинг: рассрочка",
        prev: prevArrow + "Видео",
        next: "Предложение" + nextArrow,
    },
    9: {
        id: "last-promo",
        title: "Фиброцементный сайдинг: звоните!",
        prev: prevArrow + "Рассрочка",
        next: "",
    },
}

var maxPortfolioPosition = 7;

var portfolio = {
    0: 'url(/assets/img/fibra/portfolio/0.jpg)',
    1: 'url(/assets/img/fibra/portfolio/1.jpg)',
    2: 'url(/assets/img/fibra/portfolio/2.jpg)',
    3: 'url(/assets/img/fibra/portfolio/3.jpg)',
    4: 'url(/assets/img/fibra/portfolio/4.jpg)',
    5: 'url(/assets/img/fibra/portfolio/5.jpg)',
    6: 'url(/assets/img/fibra/portfolio/6.jpg)',
    7: 'url(/assets/img/fibra/portfolio/7.jpg)',
}

File: views/fibra.go
package views

import (
	"html/template"
	"net/http"

	mobiledetect "github.com/Shaked/gomobiledetect"
	"github.com/meehalkoff/loggi"
	"gitlab.com/mvalkov/katzman/lib"
)

func FibraHandler(w http.ResponseWriter, r *http.Request) {

	tmpl := template.New("fibra")

	detect := mobiledetect.NewMobileDetect(r, nil)
	if detect.IsMobile() {
		tmpl.ParseFiles(
			"mobile/templates/fibra.html",
			"mobile/templates/header.html",
			"mobile/templates/footer.html",
			"mobile/templates/top-panel.html",
			"mobile/templates/calc.html",
			"mobile/templates/call.html",
			"mobile/templates/about.html",
			"mobile/templates/firstPromoBlock.html",
			"mobile/templates/creditPromoBlock.html",
			"mobile/templates/lastPromoBlock.html",
		)
	} else {
		tmpl.ParseFiles(
			"templates/header.html",
			"templates/footer.html",
			"templates/top-panel/top-panel.html",
			"templates/top-panel/about.html",
			"templates/top-panel/calc.html",
			"templates/top-panel/call.html",
			"templates/top-panel/main-menu.html",
			"templates/control-panel.html",
			"templates/firstPromoBlock.html",
			"templates/fibra/index.html",
			"templates/fibra/summary.html",
			"templates/fibra/palitra.html",
			"templates/fibra/kachestva.html",
			"templates/fibra/schema.html",
			"templates/fibra/otkos.html",
			"templates/fibra/portfolio.html",
			"templates/shared/video.html",
			"templates/fibra/promo.html",
			"templates/lastPromoBlock.html",
			"templates/creditPromoBlock.html",
		)
	}

	data := lib.FirstPromoBlock{
		MaterialName: `Сайдинг Фиброцемент`,
		ID:           "fibraPrice",
		PictureURL:   "/assets/img/promo/fibra.jpg",
		UnitName:     "кв.м.",
		YoutubeID:    "q0BymYA4LR4",
	}
	if err := tmpl.Execute(w, data); err != nil {
		loggi.Fatal(err)
	}
}
